src/plone/restapi/tests/test_atfield_serializer.py

```diff
@@ -70,12 +73,18 @@ def test_lines_field_serialization_returns_list(self):
         self.assertTrue(isinstance(value, list), 'Not a <list>')
         self.assertEqual([u'foo', u'bar'], value)
 
-    def test_file_field_serialization_returns_unicode(self):
+    def test_file_field_serialization_returns_dict(self):
         value = self.serialize('testFileField', 'spam and eggs',
-                               filename='spam.txt', mimetype='text/plain')
-        self.assertTrue(isinstance(value, unicode), 'Not an <unicode>')
-        self.assertEqual(u'http://nohost/plone/doc1/@@download/testFileField',
-                         value)
+                               mimetype='text/plain')
+        self.assertTrue(isinstance(value, dict), 'Not a <dict>')
+
+        url = u'http://nohost/plone/doc1/@@download/testFileField'
+        self.assertEqual(
+            {u'filename': '',
```

Comment:
We've been hit several times by the filenames (specially when running large imports), I think it's a bit obscure :wink: